shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/GettingStartedScreen.kt

```diff
@@ -93,6 +95,7 @@ fun GettingStartedScreen() {
             }
         }
         BisqButton("Create offer", onClick={ presenter.navigateToCreateOffer() })
+        BisqButton("Snack", onClick={ tabPresenter.showSnackbar("Welcome to Bisq") })
```

Comment:
this was great for testing the PR, I verified it works in all 3 apps :)

Can we remove it and add the snackbar somewhere else. E.g.: After an offer is created it coul say "Offer created"

The UI for the snackbar could be improved as it looks like a button, I think we should do a semi transparent white background and text in green or something like that. You probably know better :)

Reply:
Done